MediaGalleryUi/view/adminhtml/web/js/image/image-edit.js

```diff
@@ -48,6 +50,7 @@ define([
                 getDetails(this.imageEditDetailsUrl, [imageId]).then(function (imageDetails) {
                     this.images[imageId] = imageDetails[imageId];
                     this.image(this.images[imageId]);
+                    this.mediaGalleryEditKeywordsDetails().getKeywordsOp(this.image().tags);
```

Comment:
Instead of setting keyword options - let's export the `image` from the image-edit component to the keywords-ui-select component